### ISSUE TITLE
[BUG] Kubernetes v1.16 incompatibility

### ISSUE THREAD

**Issue by palexster** [polycube-network/polycube#213]
## Describe the bug
As found by Aldo, Kubernetes v1.16 dropped the legacy support for:
```apiVersion extensions/v1beta1```
in favor of: 
```apiVersion: apps/v1```

This breaks the Polycube CNI installation, resulting in a kubernetes failure, as the current plugin declares the compatibility with the old API and cannot be installed in K8s 1.16 and onwards.

## To Reproduce
Trying to apply polycube CNI manifest, we get:

``` $ kubectl apply -f https://raw.githubusercontent.com/polycube-network/polycube/master/src/components/k8s/standalone_etcd.yaml

error: unable to recognize "https://raw.githubusercontent.com/polycube-network/polycube/master/src/components/k8s/standalone_etcd.yaml": no matches for kind "Deployment" in version "extensions/v1beta1"
```

## Additional context

The apiVersion change is documented here: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

**Comment by SunSince90** [comment]
I have grep-ed all the files that contain the unsupported apiVersion and replaced with the new one. 
I'll make a PR when I have finished building the docker and tested it with k8s 1.16.

**Comment by alacuku** [comment]
I already have a working version of the three configuration files: pcn-k8s.yaml, standalone_etcd.yaml and echoserver_nodeport.yaml. Should I make a pull request or we wait for the Elis' fix?

**Comment by frisso** [comment]
Is Elis fix ok? If so, we can simply accept his fix and merge.

**Comment by frisso** [comment]
Closed with #214
